src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -1516,13 +1516,7 @@ public Optional<String> mainClass() {
     }
 
     /**
-     * Returns the set of packages in the module.
-     *
-     * <p> The set of packages includes all exported and open packages, as well
-     * as the packages of any service providers, and the package for the main
-     * class. </p>
-     *
-     * @return A possibly-empty unmodifiable set of the packages in the module
+     * {@return a possibly-empty unmodifiable set of all packages in the module}
```

Comment:
On second thoughts, having "possibly-empty unmodifiable set" in the first sentence of the method description will be inconsistent with all the other method descriptions. If you scan the descriptions of the other methods then you'll see they have a simple/clear first line, the "possibly-empty unmodifiable set" is in the return description. Sorry for the flip-flop, I think it will be clearer to just say "Returns the set of all packages in the module." as the first sentence.

Reply:
I've now updated the PR to follow this suggestion. The second paragraph has been removed as suggested and I haven't done any changes to the existing `@return` tag. If necessary I can adjust the text further.

This now renders as:

<img width="651" height="196" alt="packages" src="https://github.com/user-attachments/assets/60ad1584-73d1-4c71-8258-3bcb130cf916" />